concrete/src/Updater/Migrations/Migrations/Version20200609145307.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Concrete\Core\Updater\Migrations\Migrations;
+
+use Concrete\Core\Updater\Migrations\RepeatableMigrationInterface;
+use Concrete\Core\Updater\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20200609145307 extends AbstractMigration implements RepeatableMigrationInterface
+{
+
+    public function upgradeDatabase()
```

Comment:
Since it's a repeatable migration, this migration can be executed later on, when people upgrades to concrete5 v9; so it's a migration that we shouldn't be included in v9

Reply:
Yeah, I'm not merging this branch back into release/9.0.0 This is going to be the start of the 8.5.x branch that's separate from 9.0.